server/models/base/Model.ts

```diff
@@ -270,6 +270,7 @@ class Model<
       ip: context.ip,
       changes: model.previousChangeset,
       data: context.event.data,
+      createdAt: "createdAt" in model ? model.createdAt : undefined,
```

Comment:
I'm pretty sure we don't want this change – the event should reflect the time that it is recorded in the database. Better to change the tests in this case